rewrite-maven/src/main/java/org/openrewrite/maven/cache/RocksdbMavenPomCache.java

```diff
@@ -297,6 +300,31 @@ private void fillUnresolvablePoms() {
             } catch (RocksDBException exception) {
                 throw new IllegalStateException("Unable to create cache database." + exception.getMessage(), exception);
             }
+
+            try {
+                verifyCache(workspace);
+            } catch (Exception ex) {
+                throw new IllegalStateException("Unable to clear corrupt cache.", ex);
+            }
+        }
+
+        private void verifyCache(String workspace) throws IOException {
```

Comment:
The names of functions that alter state should make it explicit that they alter state.
A verb like "verify" doesn't imply change or alteration.

I suggest something like `cleanIfCorrupt` or `verifyAndClean` or something along those lines